app/controllers/discovered_hosts_controller.rb

```diff
@@ -25,6 +25,9 @@ def index
 
   def show
     # filter graph time range
+    @general= { :Highlights => ["interface", "ipaddress", "virtual", "manufacturer", "architecture", "memorysize_mb", "os", "macaddress"]}
+    @highlights ={:Network=>["discovery", "dhcp", "fqdn", "hostname", "link", "mtu", "net"], :Storage => ["blockdevice"], :Hardware => ["hardw", "manufacturer", "memo", "process"], :Software => ["bios", "os" ], :Etc => [] }
+    @facts = @host.facts_hash
     @range = nil
```

Comment:
Things I'd scratch from general pane:

* memorysize_mb: 1855.26
* virtual: physical
* is_virtual: false
* macaddress: 52:54:00:26:fd:82

Things I miss in the general pane:

processorcount: 1
physicalprocessorcount: 1
memorytotal: 1.81 GB (this is more readable)
discovery_bootif: 52:54:00:13:05:13
discovery_bootip: 192.168.122.68

and total storage size as well.